lib/analytics/page.rb

```diff
@@ -0,0 +1,72 @@
+module Analytics
+  class Page
+    def self.increment(page_id, new_member:)
+      new(page_id).increment_actions(new_member: new_member)
+    end
+
+    def initialize(page_id)
+      @page_id = page_id
+    end
+
+    def total_actions(new_members: false)
+      Analytics.store.get( key(new_members )).to_i
+    end
+
+    def total_actions_over_time(period: :hour, new_members: false)
+      send("total_by_#{period}", new_members)
+    end
+
+    def increment_actions(new_member: false)
+      if new_member
```

Comment:
I thought you could just pass `new_member` to the methods and not increment if it's false. I think I'm not totally clear on what `key` does.